examples/miscellaneous/plot_outlier_detection_bench.py

```diff
@@ -0,0 +1,157 @@
+    plt.subplot(rows, 2, dataset_idx + 1)
+
+    # loading and vectorization
+    print(f"Loading {dataset_name} data")
+    if dataset_name in ["http", "smtp", "SA", "SF"]:
+        dataset = fetch_kddcup99(
+            subset=dataset_name, percent10=True, random_state=rng
+        )
+        X = dataset.data
+        y = dataset.target
+
+    if dataset_name == "forestcover":
+        dataset = fetch_covtype()
+        X = dataset.data
+        y = dataset.target
+        idx = rng.choice(X.shape[0], int(X.shape[0]*0.1), replace=False)
+        X = X[idx, :]
+        y = y[idx]
+
+        # normal data are those with attribute 2
+        # abnormal those with attribute 4
+        s = (y == 2) + (y == 4)
+        X = X[s, :]
+        y = y[s]
+        y = (y != 2).astype(int)
+
+    print("vectorizing data")
+
+    if dataset_name == "SF":
+        idx = rng.choice(X.shape[0], int(X.shape[0]*0.1), replace=False)
+        X = X[idx, :]  # reduce the sample size
+        y = y[idx]
+        lb = LabelBinarizer()
+        x1 = lb.fit_transform(X[:, 1].astype(str))
+        X = np.c_[X[:, :1], x1, X[:, 2:]]
+        y = (y != b"normal.").astype(int)
+
+    if dataset_name == "SA":
+        idx = rng.choice(X.shape[0], int(X.shape[0]*0.1), replace=False)
+        X = X[idx, :]  # reduce the sample size
+        y = y[idx]
+        lb = LabelBinarizer()
+        x1 = lb.fit_transform(X[:, 1].astype(str))
+        x2 = lb.fit_transform(X[:, 2].astype(str))
+        x3 = lb.fit_transform(X[:, 3].astype(str))
+        X = np.c_[X[:, :1], x1, x2, x3, X[:, 4:]]
+        y = (y != b"normal.").astype(int)
+
+    if dataset_name == "http" or dataset_name == "smtp":
+        y = (y != b"normal.").astype(int)
+
+    if dataset_name in ["glass", "wdbc", "cardiotocography"]:
+        dataset = fetch_openml(name=dataset_name, version=1,
+                               as_frame=False)
+        X = dataset.data
+        y = dataset.target
+
+        if dataset_name == "glass":
+            s = y == 'tableware'
+            y = s.astype(int)
+
+        if dataset_name == "wdbc":
+            s = y == '2'
+            y = s.astype(int)
+            X_mal, y_mal = X[s], y[s]
+            X_ben, y_ben = X[~s], y[~s]
+
+            # downsampled to 39 points (9.8% outliers)
+            idx = rng.choice(
+                y_mal.shape[0], 39, replace=False)
+            X_mal2 = X_mal[idx, :]
+            y_mal2 = y_mal[idx]
+            X = np.concatenate((X_ben, X_mal2), axis=0)
+            y = np.concatenate((y_ben, y_mal2), axis=0)
+
+        if dataset_name == "cardiotocography":
+            s = y == '3'
+            y = s.astype(int)
+
+    print("Estimator processing...")
+    for model_name, model in models:
+        tstart = time()
+        model.fit(X)
+        fit_time = time() - tstart
+        if model_name == "LOF":
+            scoring = -model.negative_outlier_factor_
+            # the lower score, the more normal
```

Comment:
remove this comment add before `if model_name == "LOF":`
```python
        # use oppositve of default scoring so that the lower score, the more
        # normal
```